discord-canary.spec

```diff
@@ -29,15 +29,14 @@ mkdir -p $RPM_BUILD_ROOT/usr/share/applications
 
 cp -r * $RPM_BUILD_ROOT/opt/DiscordCanary/
 ln -sf /opt/DiscordCanary/DiscordCanary $RPM_BUILD_ROOT/%{_bindir}/
-install -m 755 Discord.desktop $RPM_BUILD_ROOT/usr/share/applications/
+desktop-file-validate %{buildroot}/%{_datadir}/applications/Discord.desktop
```

Comment:
I replaced that install command, because I was getting a similar error, of not finding he Discord.desktop file with the install